src/ray/object_manager/plasma/store.h

```diff
@@ -202,6 +203,10 @@ class PlasmaStore {
     callback(available);
   }
 
+  // NOTE(swang): This will iterate through all objects in the
+  // object store, so it should be called sparingly.
```

Comment:
It is now right? (every 10 seconds?)

Reply:
This is a new function that only gets called on OOM.